core/services/feeds/service.go

```diff
@@ -421,14 +420,6 @@ func (s *service) UpdateChainConfig(ctx context.Context, cfg ChainConfig) (int64
 	return id, nil
 }
 
-// Lists all JobProposals
-//
-// When we support multiple feed managers, we will need to change this to filter
-// by feeds manager
-func (s *service) ListJobProposals(ctx context.Context) ([]JobProposal, error) {
```

Comment:
Who called this method? Shouldn't the call site be replaced with the ListJobProposalsByManagerID? Unless I missed that.